# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/afbinit/hotfix-cc.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- afbinit-1.0.orig/Makefile.vanilla	2022-07-18 16:35:35.117242809 +0200
+++ afbinit-1.0.orig/Makefile	2022-07-18 16:35:41.384242559 +0200
@@ -1,6 +1,6 @@
 # $Id: Makefile,v 1.1.1.1 1999/09/08 06:42:36 davem Exp $
 afbinit: afbinit.c
-	gcc -Wa,-Av9a -O -o afbinit afbinit.c
+	$(CC) -Wa,-Av9a -O -o afbinit afbinit.c
 
 clean:
 	rm -f afbinit afbinit.o
